src/components/Home.vue

```diff
@@ -541,7 +541,7 @@ export default {
                     }
                 }
             }, respError => {
-                Vue.set(this.balance, address, new BigNumber(0))
+                Vue.set(this.balance, address, new BigNumber(NaN))
```

Comment:
Move this line before calling `this.chain.getBalanceDetail(address)` (Line 530). The client network might very lag (It sometimes spends more than 10 seconds to get the response). If user switch account quickly, he would see last address balance in current selected address.